code/datums/components/crafting/recipes/recipes_primal.dm

```diff
@@ -109,11 +109,47 @@
 				 /obj/item/stack/sheet/sinew = 4)
 	category = CAT_PRIMAL
 
+/datum/crafting_recipe/mushroom_bowl
+	name = "Mushroom Bowl"
+	result = /obj/item/reagent_containers/glass/bowl/mushroom_bowl
+	reqs = list(/obj/item/reagent_containers/food/snacks/grown/ash_flora/shavings = 5)
+	time = 30
+	category = CAT_PRIMAL
+
+/datum/crafting_recipe/ashen_stablizer
+	name = "Herbal Stabilizing Bowl"
+	result = /obj/item/hivelordstabilizer/ashen
+	parts = /obj/item/reagent_containers/glass/bowl/mushroom_bowl = 1
```

Comment:
make this a req instead, not compiling because of this

Reply:
Thats not why it failed